pinwheel-android/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
     id 'maven-publish'
 }
 
-def libraryVersion = '2.3.6'
+def libraryVersion = '2.3.7'
```

Comment:
Can't add a comment there but on line 23 versionCode needs to be bumped

Reply:
good catch, thank you!